src/Couchbase.Lite.Shared/API/Info/Defaults.cs

```diff
@@ -78,10 +78,10 @@ public static class Constants {
 		public static readonly int DefaultReplicatorMaxAttemptsSingleShot = 10;
 
 		/// <summary>
-		/// Default value for <see cref="ReplicatorConfiguration.MaxAttempts" /> (-1)
+		/// Default value for <see cref="ReplicatorConfiguration.MaxAttempts" /> (Int32.MaxValue)
 		/// When replicator is continuous, never give up unless explicitly stopped
 		/// </summary>
```

Comment:
I think this could be a value from Defaults

Reply:
Not sure what you mean.  It is.